src/modes/shop/shop_trade.cpp

```diff
@@ -428,19 +432,19 @@ bool TradeInterface::_ChangeCategory(bool left_or_right)
         return false;
 
     if(left_or_right == false) {
-        _current_category = (_current_category == 0) ? (_number_categories - 1) : (_current_category - 1);
+        _current_category = (_current_category == 0)
+          ? (_number_categories - 1) : (_current_category - 1);
```

Comment:
I'm used to keep the '?' at the same line than the condition to make it look like a question, which is more readable IMHO. Could you change that? Thanks.